src/styles/platform-api-keys.css

```
@@ -0,0 +1,135 @@
+}
+
+.platform-api-keys-page .app-table tbody tr:last-child .app-td {
+    border-bottom: none;
+}
+
+.platform-api-keys-page .app-table tbody tr:hover .app-td {
+    background-color: var(--secondary-bg-color, #f8f9fa);
+}
+
+.platform-api-keys-page .masked-api-key {
+    font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, monospace;
```

Stylelint (17.5.0)

[error] Expected quotes around "Liberation Mono" (font-family-name-quotes)

(font-family-name-quotes)